From: Vivek Goyal <vgoyal@redhat.com>
Date: Fri, 4 Dec 2009 10:36:43 -0500
Subject: [PATCH] blkio: Allow CFQ group IO scheduling even when CFQ is a module
X-Git: 846954b0a32f4ae953e082eabd178e7a98dd2efd
References: FATE#311054
Patch-Mainline: 2.6.33

o Now issues of blkio controller and CFQ in module mode should be fixed.
  Enable the cfq group scheduling support in module mode.

Signed-off-by: Vivek Goyal <vgoyal@redhat.com>
Signed-off-by: Jens Axboe <jens.axboe@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 block/Kconfig.iosched |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/block/Kconfig.iosched b/block/Kconfig.iosched
index 5368d74..b71abfb 100644
--- a/block/Kconfig.iosched
+++ b/block/Kconfig.iosched
@@ -34,7 +34,7 @@ config IOSCHED_CFQ
 
 config CFQ_GROUP_IOSCHED
 	bool "CFQ Group Scheduling support"
-	depends on IOSCHED_CFQ=y && CGROUPS
+	depends on IOSCHED_CFQ && CGROUPS
 	select BLK_CGROUP
 	default n
 	---help---
-- 
1.6.0.2

